Sources/StripeKit/Core Resources/Charges/ChargeRoutes.swift

```diff
@@ -391,6 +391,6 @@ public struct StripeChargeRoutes: ChargeRoutes {
             queryParams = filter.queryParameters
         }
         
-        return apiHandler.send(method: .GET, path: charges, query: queryParams, headers: headers)
+        return apiHandler.send(method: .GET, path: String(charges.dropLast()), query: queryParams, headers: headers)
```

Comment:
@apps4everyone This can be just `charge` checkout line 202.

Reply:
changed the PR